docs/howtos.rst

```diff
@@ -0,0 +1,53 @@
+Flax HOWTOs
+===========
+
+Flax aims to be a thin library of composable primitives. You compose
+these primites yourself into a training loop that you write and is fully under
+your control. You have full freedom to modify the behavior of your training loop,
+and you should generally not need special library support to implement
+the modifications you want.
+
+To help you get started, we show some sample diffs, which
```

Comment:
This newline seems wrong, will it render correctly?

Reply:
I'm not sure what newline you mean. Is it that the line is shorter than 80c? Single newlines don't have any meaning in restructuredText. But two newlines in a row are a paragraph delineator?

Reply:
Indeed I meant the newline on the line that is shorter than 80c. SG.